datadog_checks_dev/datadog_checks/dev/tooling/git.py

```diff
@@ -39,13 +39,20 @@ def files_changed():
     Return the list of file changed in the current branch compared to `master`
     """
     with chdir(get_root()):
-        result = run_command('git diff --name-status master...', capture='out')
-    status_lines = result.stdout.splitlines()
+        # Use `--name-status` to include moved files
+        name_status_result = run_command('git diff --name-status master...', capture='out')
+        # Use `--name-only` to include uncommitted files
```

Comment:
Would be helpful to have an arg to make including these files optional and explicitly part of the results.

Reply:
@mgarabed I think this might be added latter if needed since I'm not sure about the use case (YAGNI?). 

If we do not want to include uncommitted files we can still stash the uncommitted files for example, but do we have a real use case for that?

Reply:
YAGNI means nothing if the feature is merely 2-3 lines... 😉

Reply:
@mgarabed @ofek I just added `include_uncommitted=True`. Let me know if that's ok for you :)